spring-aot-test/src/main/java/org/springframework/aot/test/context/bootstrap/generator/TestContextAotProcessor.java

```diff
@@ -126,6 +132,34 @@ protected ClassName generateTestContext(BootstrapWriterContext writerContext, Su
 				.addModifiers(Modifier.PUBLIC, Modifier.STATIC).addCode(code.build());
 	}
 
+	private MethodSpec.Builder getContextInitializersBuilder(Map<ClassName, TestContextConfigurationDescriptor> entries) {
+		// We're generating a method that looks like the following.
+		//
+		// public static Map<String, Class<? extends ApplicationContextInitializer<?>>> getContextInitializers() {
+		//     Map<String, Class<? extends ApplicationContextInitializer<?>>> map = new HashMap<>();
+		//     map.put("org.example.Sample1Tests", Sample1TestsContextInitializer.class);
+		//     map.put("org.example.Sample2Tests", Sample2TestsContextInitializer.class);
+		//     return map;
+		// }
+
+		ClassName stringTypeName = ClassName.get(String.class);
+		TypeName initializerWildcard = WildcardTypeName.subtypeOf(Object.class);
+		TypeName initializerTypeName = ParameterizedTypeName.get(ClassName.get(ApplicationContextInitializer.class), initializerWildcard);
+		TypeName classWildcard = WildcardTypeName.subtypeOf(initializerTypeName);
+		TypeName classTypeName = ParameterizedTypeName.get(ClassName.get(Class.class), classWildcard);
+		TypeName mapTypeName = ParameterizedTypeName.get(ClassName.get(Map.class), stringTypeName, classTypeName);
+
+		Builder code = CodeBlock.builder();
+		code.addStatement("$T map = new $T<>()", mapTypeName, HashMap.class);
```

Comment:
Detail but the field is called `entries` in the other method, why use something else here?

Reply:
Speaking of "field", I actually was wondering about converting it from a local variable to a `static` field, since we're currently recreating the maps each time one of these methods is invoked (which happens a lot for `getContextInitializers()`).

Had you considered that?

Was there a technical reason to create the map as a local variable instead of as a static field?

Reply:
With regard to the name of the variable, every map contains entries. So unless the term "entries" has special meaning for the context in which a map is used, I always prefer the generic term `map` unless I think it's necessary to name the map something  more specific.

Since this is (currently) a local variable, I don't see the need for anything more specific than `map`.

With regard to using something different here, I didn't want to change the implementation of the other method, because I know you do not like changes that you do not consider relevant to the current PR.

Reply:
> With regard to using something different here, I didn't want to change the implementation of the other method, because I know you do not like changes that you do not consider relevant to the current PR.

That's exactly the point that I am trying to raise for several PRs in a row now. If you disagree about something, that's fine to discuss it and then come to a conclusion but changing it in your own code just brings inconsistency and I don't like that.

Reply:
> Speaking of "field", I actually was wondering about converting it from a local variable to a static field, since we're currently recreating the maps each time one of these methods is invoked (which happens a lot for getContextInitializers()).

You can disregard that. I think that used to be the case, but it no longer is. The methods that create the maps should only be invoked twice now: once for `AotCacheAwareContextLoaderDelegate` and once for `AotDependencyInjectionTestExecutionListener`.